services/dynamic-scheduler/tests/unit/api_rpc/test_api_rpc__services.py

```diff
@@ -354,6 +358,7 @@ async def test_stop_dynamic_service(
         node_id=node_id,
         simcore_user_agent=simcore_user_agent,
         save_state=save_state,
+        primary_group_id=1,
```

Comment:
the groupID 1 is usually the everyone group. I would try to not use it in tests. you might get tricked by that fact...
Also, I think we might have fixtures for creating group ids...